lib/launcher.js

```diff
@@ -2,6 +2,7 @@
 
 var net = require('net');
 var portfile = require('../lib/portfile');
+var process = require('process');
```

Comment:
You don’t need to require `process`. It’s a global 😉

Reply:
Oops, fixed and thanks.